src/utils/middlewares/azure_api_auth.ts

```diff
@@ -80,7 +80,10 @@ export enum UserGroup {
   ApiMessageWriteEUCovidCert = "ApiMessageWriteEUCovidCert",
 
   // messages: send messages with payee
-  ApiMessageWriteWithPayee = "ApiMessageWriteWithPayee"
+  ApiMessageWriteWithPayee = "ApiMessageWriteWithPayee",
+
+  // messages: send messages with legal data
+  ApiMessageWriteWithLegal = "ApiMessageWriteWithLegal"
```

Comment:
Do we have constraints on label's lenght? if no, should we go for `ApiMessageWriteWithLegalData`=

Reply:
According with Azure doc, max length is 128 chars, so we can add the data prefix.
(https://docs.microsoft.com/en-us/azure/role-based-access-control/custom-roles)